src/styles/main.scss

```
@@ -1,3 +1,62 @@
-body {
-  margin: 0;
+.calendar {
+  border: 1px solid black;
+
+  height: 505px;
+  width: 707px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  padding: 10px;
+
+  &__days {
+    position: relative;
+    box-sizing: border-box;
+    height: 100px;
+    width: 100px;
+    background-color: #eee;
+    border: 1px solid black;
+
+    cursor: pointer;
+    transition: 500ms;
```

```suggestion
    transition: background-color 500ms, ...;
```
Specify transition properties